### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by vidhill** [ergo/polymer-ui-router#4]
Hi, 

I'm just wondering where this implementation stands maturity wise, would you recommend using this on a production app? 

I'm finding app-router & iron-pages very frustrating and I'd like to find a better solution.. I've used ui-router with angular and found it great.

Is your crossroadsjs implementation further along/more stable?

**Comment by ergo** [comment]
Hi @vidhill, sorry for late reply.

Def. `polymer-ui-router` is the way to go - I used it in production without issues.

As for `crossroadsjs` -  I also used it in production and while it does work, it turned out that it is unmaintained by the author and the project has some unfixed bugs related to route generation for more complex cases. I abandoned development - just not worth the hassle.

`polymer-ui-router` seems to work just fine and `ui-router-core` used by this component is maintained by Angular community, and feature wise it is one of the best routers in JS world in my opinion.
